app/controllers/application.js

```diff
@@ -1,29 +1,27 @@
 import Ember from 'ember';
 
 export default Ember.Controller.extend({
-  breadCrumb: 'Home',
-
-	isExpanded: true,
-	isNotLogin: Ember.computed('this.currentPath', function() {
-	  	if (this.currentPath !== 'login') {
-	  		return true;
-	  	} else {
-	  		return false;
-	  	}
-	}),
-	sizeContainer: function() {
+  isExpanded: true,
+  isNotLogin: Ember.computed('this.currentPath', function() {
+      if (this.currentPath !== 'login') {
+        return true;
+      } else {
+        return false;
+      }
+  }),
+  sizeContainer: function() {
         var winWidth = $(window).width();
         if (winWidth < 992 && this.isExpanded) {
-        	this.send('toggleMenu');
+          this.send('toggleMenu');
```

Comment:
We're using 4 spaces for indentation, not 2. Chris was wrong we he reformatted this.